### PR TITLE
[Logs] Add an endpoint to modify log levels for log rules.

### PR DESCRIPTION
**Pull request by bokobza**
I added an endpoint to modify log levels on the fly.

Here's a payload example:
`PUT /api/Node/loglevels`

```
{
  "logRules": [
    {
      "ruleName": "Stratis.Bi.Features.Consensus.*",
      "logLevel": "Critical"
    },
    {
      "ruleName": "Stratis.Bitcoin.Features.Api.*",
      "logLevel": "Trace"
    },
  ]
}
```

### CODE REVIEW

File: src/Stratis.Bitcoin/Controllers/NodeController.cs [hunk 1]
```diff
@@ -399,6 +404,61 @@ public IActionResult Shutdown([FromBody] bool corsProtection = true)
...
+                            rule.EnableLoggingForLevel(level);
+                        }
+                        else
+                        {
+                            rule.DisableLoggingForLevel(level);
+                        }
+                    }
+                }
+
+                // Only update the loggers if the setting was succesfull.
```
Comment (noescape00) [inline src/Stratis.Bitcoin/Controllers/NodeController.cs]: `successful`

Comment (bokobza) [reply src/Stratis.Bitcoin/Controllers/NodeController.cs]: eh? I changed it yesterday
image

where is it?

Comment (bokobza) [reply src/Stratis.Bitcoin/Controllers/NodeController.cs]: ah look it's up there ⏫ 
I forgot to push it, but when I did just now, it appeared before your comment. 
I guess the sorting is done by time.

Comment (noescape00) [reply src/Stratis.Bitcoin/Controllers/NodeController.cs]: I guess github is being weird (or I am)

**Comment by bokobza** [comment]
Ran the MacOS again, and it was successful this time.
image

**Pull request merged**